playbooks/robusta_playbooks/node_enrichments.py

```diff
@@ -100,20 +106,13 @@ def node_health_watcher(event: NodeChangeEvent):
     old_condition = [c for c in event.old_obj.status.conditions if c.type == "Ready"]
 
     if len(new_condition) != 1 or len(old_condition) != 1:
-        logging.warning(
-            f"more than one Ready condition. new={new_condition} old={old_condition}"
-        )
+        logging.warning(f"more than one Ready condition. new={new_condition} old={old_condition}")
 
-    new_condition = new_condition[0]
-    old_condition = old_condition[0]
-
-    currently_ready = "true" in new_condition.status.lower()
-    previously_ready = "true" in old_condition.status.lower()
+    currently_ready = "true" in new_condition[0].status.lower()
+    previously_ready = "true" in old_condition[0].status.lower()
```

Comment:
im just curious if this was automatic or you did this?

Reply:
No, I think I did it manually but I don't remember why I commited it actually
I think because I just thought it looks better, or maybe flake8 didn't like something